Sources/W3CTraceContext/SpanID.swift

```diff
@@ -16,27 +16,31 @@
 ///
 /// [W3C TraceContext: parent-id](https://www.w3.org/TR/trace-context-1/#parent-id)
 public struct SpanID: Sendable {
-    private let _bytes: Bytes
-
-    /// An 8-byte array representation of the span ID.
-    public var bytes: [UInt8] {
-        withUnsafeBytes(of: _bytes, Array.init)
-    }
+    /// The 8 bytes making up the span ID.
+    public let bytes: Bytes
 
     /// Create a span ID from 8 bytes.
     ///
     /// - Parameter bytes: The 8 bytes making up the span ID.
     public init(bytes: Bytes) {
-        _bytes = bytes
+        self.bytes = bytes
     }
 
+    @inlinable
+    public func withUnsafeBytes<Result>(_ body: (UnsafeRawBufferPointer) throws -> Result) rethrows -> Result {
+        try bytes.withUnsafeBytes(body)
+    }
+
+    /// A null span ID.
+    public static var null: SpanID { SpanID(bytes: .null) }
```

Comment:
```suggestion
    /// An invalid span ID with all bytes set to zero.
    public static var invalid: SpanID { SpanID(bytes: .invalid) }
```

An all-zero span/trace ID is called `invalid` in the W3C Trace Context spec and other OTel client libraries, so I'd follow that naming pattern here too.

Reply:
Maybe on the `Bytes` it should be called `.null` but on the ID it should be called `.invalid`?

Reply:
Yep, works for me 👍